src/multinode_run.jl

```diff
@@ -380,47 +375,43 @@ end
 """
 Initialize elbo args for the specified target source.
 """
-function init_elboargs(ts::Int, catalog::Vector{CatalogEntry},
+function init_elboargs(config::Configs.Config,
+                       ts::Int,
+                       catalog::Vector{CatalogEntry},
                        target_sources::Vector{Int},
-                       neighbor_map::Vector{Vector{Int}}, images::Vector{Image},
-                       ts_vp::Dict{Int64,Array{Float64}},
+                       neighbor_map::Vector{Vector{Int}},
+                       images::Vector{Image},
                        ea_vec::Vector{ElboArgs},
-                       cfg_vec::Vector{Config{DEFAULT_CHUNK,Float64}};
-                       use_fft=false,
-                       trust_region=NewtonTrustRegion())
-    entry_id = target_sources[ts]
-    entry = catalog[entry_id]
-    neighbor_ids = neighbor_map[ts]
-    neighbors = catalog[neighbor_ids]
-    cat_local = vcat([entry], neighbors)
-    ids_local = vcat([entry_id], neighbor_ids)
+                       vp_vec::Vector{VariationalParams{Float64}},
+                       cfg_vec::Vector{Config{DEFAULT_CHUNK,Float64}},
+                       ts_vp::Dict{Int64,Array{Float64}})
     try
+        entry_id = target_sources[ts]
+        entry = catalog[entry_id]
+        neighbor_ids = neighbor_map[ts]
+        neighbors = catalog[neighbor_ids]
+        cat_local = vcat([entry], neighbors)
+        ids_local = vcat([entry_id], neighbor_ids)
+
         patches = Infer.get_sky_patches(images, cat_local)
-        Infer.load_active_pixels!(images, patches)
+        Infer.load_active_pixels!(config, images, patches)
```

Comment:
This code is duplicated in `initialize_elboargs_sources_kernel!`. Would you make the shared parts a new function in `joint_infer.jl`? That way it'll be exercised by the unit tests and there won't be divergence between the two versions.